frontend/src/react/Exam/NextButton.js

```diff
@@ -0,0 +1,8 @@
+import React,{Component} from 'react';
```

Comment:
Can you delete this component ? It was refactored somewhere else

Reply:
Done @ValentinChion

Reply:
Great job otherwise friend :rocket: